[docs] workaround for nextjs (#199)
nylas/components: README.md

</main>
```

> ## Workarounds

Reviewer: ```suggestion
## Framework-specific workarounds
```

> ## Workarounds

Reviewer 2: Not sure "workaround" is the right term here, since there isn't really a bug to work around. Maybe this section title can just be a catch-all for server-side rendering implementations? 
```suggestion
## Server-side Rendering
```

> ### NextJS

Reviewer: Toss a link to https://nextjs.org/ somewhere in this section IMO

> `nylas/components` leverages web components so for any server-side framework, we should import components at runtime.

Reviewer 2: ```suggestion
`nylas/components` leverages web components so for any server-side framework, components should be imported at runtime.
```
